src/manage_nsfs/manage_nsfs_cli_responses.js

```diff
@@ -131,16 +148,19 @@ ManageCLIResponse.LoggingExported = Object.freeze({
 
 ManageCLIResponse.UpgradeSuccessful = Object.freeze({
     code: 'UpgradeSuccessful',
+    message: 'Upgrade completed successfully',
```

Comment:
Can you add to all the Upgrade functions Config Directory Upgrade prefix? for example - Config Directory Upgrade completed successfully etc

Reply:
Sure